server/client_manager.py

```diff
@@ -118,16 +121,25 @@ def send_command(self, command: str, *args):
 
             Args:
                 command (str): command name
-                *args: list of arguments
+                *args: tuple containing the packet arguments
             """
             if args:
                 if command == 'MS':
                     for evi_num in range(len(self.evi_list)):
                         if self.evi_list[evi_num] == args[11]:
-                            lst = list(args)
+                            lst = list(args) # convert to a list so we can modify it
                             lst[11] = evi_num
-                            args = tuple(lst)
+                            args = tuple(lst) # then convert back to a tuple when finished
                             break
+                        # <2.9 can't parse Y offset so we strip it out based on version
+                        if self.version.major == '2' and self.version.minor in ['8', '7', '6']:
```

Comment:
You forgot to change this
```suggestion
                        if self.release == '2' and self.major_version in ['8', '7', '6']:
```